hack/setup-kind-in-ci.sh

```diff
@@ -373,6 +369,10 @@ setup_kind_multicluster() {
     mkdir -p "${certs_dir}"/keycloak
   fi
 
+  if [[ -n "${KIND_NODE_IMAGE}" ]]; then
```

Comment:
not sure why the double-brackets are for... `-n` test need only single bracket
```suggestion
  if [ -n "${KIND_NODE_IMAGE}" ]; then
```